xml/System.Activities.Presentation.PropertyEditing/EditModeSwitchButton.xml

```diff
@@ -14,7 +14,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>Represents a button that enables the user to switch between the different <see cref="T:System.Activities.Presentation.PropertyEditing.PropertyContainerEditMode" /> modes within the property grid .</summary>
+    <summary>Represents a button that enables the user to switch between the different PropertyContainer modes within the property grid .</summary>
```

Comment:
I think that the original version was preferable; PropertyContainer modes is unclear. The original link also wasn't broken.

Reply:
But I made that change to fix that link but I noticed that the /// comments refer to PropertyContainer modes. The problem is that PropertyContainer is not public,